MiniAn/minian_main.py

```diff
@@ -467,7 +470,113 @@ def load_doric_to_xarray(
 
     return varr, file_
 
-
+            
+def cross_register(AC, A, minian_parameters):
+    perform_cross_reg = minian_parameters.params_cross_reg["crossReg"]
+    if not perform_cross_reg:
+        return A
+    param_dist = 5 # Defines the max dist between centroids on different sessions to consider them as same cell
+    ref_filename = minian_parameters.params_cross_reg["fname"]
+    ref_images   = minian_parameters.params_cross_reg["h5path_images"]
+    ref_ROIs     = minian_parameters.params_cross_reg["h5path_roi"]
+
+    # Load base/reference file to xarray
+    ref, file_ref_ =  load_doric_to_xarray(ref_filename, ref_images, np.float64, None, "subset", None)
+
+    # max project of the miniAn images for base file and current file
+    ref_max     = ref.max("frame")
+    AC_max = AC.max("frame")
+
+    concatenated_maxAC = xr.concat([ref_max, AC_max], pd.Index(["session1", "session2"], name = "session"))
+    # temps = result.rename('temps')
```

Comment:
Should it be removed? Could you make sure that the whole code is clean (indentations, spaces, comments, variable names, etc.)?